packages/react/src/components/TextHighlighter/__tests__/TextHighlighter.test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import '@testing-library/jest-dom';
+
+import { TextHighlighter } from '../components/TextHighlighter';
+jest.mock('./button.scss', () => ({}));
```

Comment:
@punnoosewilson can you add tests for the component here? Otherwise the ci-check will continue to fail

Reply:
Nvm, there was discussion around this and for components in draft status we can have a simple test case so it isn't a blocker